geoprocessing/src/modules/surface-cost/surface-cost.module.ts

```diff
@@ -9,14 +10,27 @@ import { PuExtractorPort } from './ports/pu-extractor/pu-extractor.port';
 import { ArePuidsAllowedPort } from './ports/pu-validator/are-puuids-allowed.port';
 import { ShapefileConverterPort } from './ports/shapefile-converter/shapefile-converter.port';
 
+import { TypeormCostSurface } from './adapters/typeorm-cost-surface';
+import { ScenariosPuCostDataGeo } from '../scenarios/scenarios-pu-cost-data.geo.entity';
+import { ScenariosPlanningUnitGeoEntity } from '../scenarios/scenarios-planning-unit.geo.entity';
+
 @Module({
-  imports: [WorkerModule],
+  imports: [
+    WorkerModule,
+    TypeOrmModule.forFeature([
+      ScenariosPuCostDataGeo,
+      ScenariosPlanningUnitGeoEntity, // not used but has to imported somewhere
+    ]),
+  ],
   providers: [
     SurfaceCostWorker,
-    SurfaceCostProcessor,
+    {
+      provide: WorkerProcessor,
+      useClass: SurfaceCostProcessor,
```

Comment:
fix of https://github.com/Vizzuality/marxan-cloud/pull/211 - otherwise module couldn't be used